drivers/at90can128/canard_at90can128.h

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2016 UAVCAN Team
+ *
+ * Distributed under the MIT License, available in the file LICENSE.
+ *
+ */
+
+#ifndef CANARD_AVR_H
+#define CANARD_AVR_H
+
+#include <canard.h>
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+
+/**
+ * Initializes the AVR instance.
+ */
+int canardAVRInit(uint32_t id);
+
+/**
+ * Deinitializes the AVR instance.
+ */
+int canardAVRClose(void);
+
+/**
+ * Transmits a CanardCANFrame to the CAN device.
+ */
+int canardAVRTransmit(const CanardCANFrame* frame);
+
+/**
+ * Receives a CanardCANFrame from the CAN device.
```

Comment:
I suggest to explicitly document return values, and that the function is non-blocking. The TX function could also use some documenting. Thanks.